manifests/mod/log_forensic.pp

```diff
@@ -0,0 +1,9 @@
+# @summary
+#   Installs `mod_log_forensic`
+# 
+# @see https://httpd.apache.org/docs/current/mod/mod_log_forensic.html for additional documentation.
+#
+class apache::mod::log_forensic {
+  include apache
+  ::apache::mod { 'log_forensic': }
```

Comment:
```suggestion
  apache::mod { 'log_forensic': }
```

This legacy syntax is from the Puppet 3 era IIRC.  The module use it a lot, but maybe we can avoid it in new code?